core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -1,13 +1,38 @@
 {% extends "@Contao/content_element/_base.html.twig" %}
 {% use "@Contao/component/_stylesheet.html.twig" %}
 
+{#
+Global settings for all sliders
+- Create a new template 'swiper.html.twig' (content_element/swiper.html.twig)
+- Copy the block ‘init_options’ and customise them (https://swiperjs.com/)
+
+Custom template
+- Create folder 'swiper' if it does not exist
+- Create a new template 'custom.html.twig' in folder 'swiper' (content_element/swiper/custom.html.twig)
+- Copy the section ‘Slider settings’ and customise them (https://swiperjs.com/)
+- After the section ‘Slider settings’, assign the new setting to ‘slider_attributes’:
+
+{% set slider_attributes = attrs().set('data-settings', slider_settings|json_encode) %}
+
+#}
+
+{# Slider settings #}
+
+{% set slider_settings = {
+    speed: speed,
+    initialSlide: offset,
+    loop: loop,
+} %}
+{% if delay > 0 %}
+    {% set slider_settings = slider_settings|merge({ autoplay: {delay: delay} }) %} 
+{% endif %}
+
+{# End Slider settings #}
+
 {% block content %}
     {% set slider_attributes = attrs()
         .addClass('swiper')
-        .set('data-delay', delay)
-        .set('data-speed', speed)
-        .set('data-offset', offset)
-        .set('data-loop', loop)
+        .set('data-settings', slider_settings|json_encode )
```

Comment:
```suggestion
        .set('data-settings', slider_settings|json_encode)
        .set('data-delay', delay)
        .set('data-speed', speed)
        .set('data-offset', offset)
        .set('data-loop', loop)
```

The old data attributes should be kept for backwards compatibility. If someone already customized their swiper template, everything will still work then.